provisioning/plugins/corim-psa-decoder/test-harness/build-test-vectors.sh

```diff
@@ -7,15 +7,15 @@ set -o pipefail
 
 # function generate_go_test_vector constructs CBOR test vector using
 # supplied comid and corim json template and saves them in a file
-# $1 file name for comid json template
-# $2 file name for corim json template
+# $1 file name for comid json template, example one of COMID_TEMPLATES
+# $2 file name for corim json template, example CORIM_CCA_TEMPLATE
 # $3 a qualifier for each cbor test vector name
 # $4 name of the file where the generated CBOR test vectors are aggregated
 generate_go_test_vector () {
 	echo "generating test vector using $1 $2"
 	cocli comid create -t ${1}.json
 	cocli corim create -m ${1}.cbor -t $2 -o corim${1}.cbor
```

Comment:
any reason why sometimes it's `${1}` and some other `$1` ?

Reply:
ok, I have corrected in the new commit now!